dev-cpp/gtkmm/gtkmm-2.24.5.ebuild

```diff
@@ -22,11 +22,8 @@ COMMON_DEPEND="
 	>=dev-cpp/pangomm-2.34.0:1.4[${MULTILIB_USEDEP}]
 	>=dev-libs/libsigc++-2.3.2:2[${MULTILIB_USEDEP}]
 "
-RDEPEND="${COMMON_DEPEND}
-	abi_x86_32? (
-		!<=app-emulation/emul-linux-x86-gtkmmlibs-20140508
-		!app-emulation/emul-linux-x86-gtkmmlibs[-abi_x86_32(-)] )
-"
+RDEPEND="${COMMON_DEPEND}"
```

Comment:
You can now get rid of COMMON_DEPEND

Reply:
K, I was just trying to modify it as little as possible and otherwise keep the 'spirit' of the
ebuild.